File: PAT-GPLT/L1008.java
package gplt;
import java.util.*;
public class L1008 {
	public static void main(String[] args){
		Scanner sc=new Scanner(System.in);
		int a=sc.nextInt();
		int b=sc.nextInt();
		int sum=0;
		int count=0;
		for(int i=a;i<=b;i++){
			System.out.printf("%5d",i);
			sum+=i;
			if(++count%5==0) System.out.println();
		}
		System.out.println("\nSum = "+sum);
		sc.close();
	}

}

File: PAT-GPLT/L1008.cpp
#include <iostream>
using namespace std;
int main(){
	int a,b,sum=0;
	cin>>a>>b;
	for(int i=a,j=0;i<=b;i++){
		sum+=i;
		printf("%5d",i);
		if(++j%5==0) cout<<endl;
	}
	if((b-a+1)%5!=0) cout<<endl;
	cout<<"Sum = "<<sum;
	return 0;
}
